media-tv/plex-media-server/plex-media-server-1.13.9.5456.ebuild

```diff
@@ -0,0 +1,135 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit user systemd unpacker python-single-r1
+
+DESCRIPTION="A free media library that is intended for use with a plex client."
+HOMEPAGE="http://www.plex.tv/"
+
+COMMIT="ecd600442"
+
+_APPNAME="plexmediaserver"
+_USERNAME="plex"
+_SHORTNAME="${_USERNAME}"
+_FULL_VERSION="${PV}-${COMMIT}"
+
+URI="https://downloads.plex.tv/plex-media-server"
+SRC_URI="
+	amd64? ( ${URI}/${_FULL_VERSION}/plexmediaserver_${_FULL_VERSION}_amd64.deb )
+	x86? ( ${URI}/${_FULL_VERSION}/plexmediaserver_${_FULL_VERSION}_i386.deb )"
+
+LICENSE="Plex"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+RESTRICT="mirror bindist strip"
+IUSE="avahi"
+
+DEPEND="sys-apps/fix-gnustack"
+RDEPEND="avahi? ( "net-dns/avahi" )"
+
+EXECSTACKED_BINS=( "${ED%/}/usr/lib/plexmediaserver/libgnsdk_dsp.so*" )
+
+QA_DESKTOP_FILE="usr/share/applications/plexmediamanager.desktop"
+QA_PREBUILT="*"
+QA_MULTILIB_PATHS=(
+	"usr/lib/${_APPNAME}/.*"
+	"usr/lib/${_APPNAME}/Resources/Python/lib/python2.7/.*"
+)
+
+S="${WORKDIR}"
+
+pkg_setup() {
+	enewgroup ${_USERNAME}
+	enewuser ${_USERNAME} -1 /bin/bash /var/lib/${_APPNAME} "${_USERNAME},video"
+
+	python-single-r1_pkg_setup
+}
+
+src_unpack() {
+	unpack_deb ${A}
+}
+
+src_prepare() {
+	eapply "${FILESDIR}"/plexmediamanager.desktop.patch
+	default
+}
+
+src_install() {
+	# Move the config to the correct place
+	local CONFIG_VANILLA="/etc/default/"${_APPNAME}""
+	local CONFIG_PATH="/etc/${_SHORTNAME}"
+	dodir "${CONFIG_PATH}"
+	insinto "${CONFIG_PATH}"
+	doins "${CONFIG_VANILLA#/}"
+	sed -e "s#${CONFIG_VANILLA}#${CONFIG_PATH}/$(basename "${CONFIG_VANILLA}")#g" \
+		-i "${S}"/usr/sbin/start_pms || die
+
+	# Remove Debian specific files
+	rm -rf "usr/share/doc" || die
+
+	# Copy main files over to image and preserve permissions so it is portable
+	cp -rp usr/ "${ED}" || die
+
+	# Make sure the logging directory is created
+	local LOGGING_DIR="/var/log/pms"
+	dodir "${LOGGING_DIR}"
+	chown "${_USERNAME}":"${_USERNAME}" "${ED%/}/${LOGGING_DIR}" || die
+
+	# Create default library folder with correct permissions
+	local DEFAULT_LIBRARY_DIR="/var/lib/${_APPNAME}"
+	dodir "${DEFAULT_LIBRARY_DIR}"
+	chown "${_USERNAME}":"${_USERNAME}" "${ED%/}/${DEFAULT_LIBRARY_DIR}"
+
+	# Install the OpenRC init/conf files depending on avahi.
+	if use avahi; then
+		doinitd "${FILESDIR}/init.d/${PN}"
+	else
+		cp "${FILESDIR}/init.d/${PN}" "${S}/${PN}" || die;
+		sed -e '/depend/ s/^#*/#/' -i "${S}/${PN}"
```

Comment:
I see you are editing one file, it doesn't need multiple sed commands. You can use this syntax: 
```
sed -i \
		-e 'edits \
		-e 'edits' \
		-e 'edits' \
		filename.txt || die
```